plugins/BTagAnalyzer.cc

```diff
@@ -2757,23 +2758,23 @@ void BTagAnalyzerT<IPTI,VTX>::processJets(const edm::Handle<PatJetCollection>& j
     float CvsLPos = pjet->bDiscriminator(CvsLPosCJetTags_.c_str());
 
     // Jet information
-		//DeepFlavour discriminators
-    JetInfo[iJetColl].Jet_DeepFlavourBDisc[JetInfo[iJetColl].nJet]     = DeepFlavourB + DeepFlavourBB + DeepFlavourLepB;
-    JetInfo[iJetColl].Jet_DeepFlavourCvsLDisc[JetInfo[iJetColl].nJet]  = DeepFlavourC/(DeepFlavourC + DeepFlavourUDS + DeepFlavourG);
-    JetInfo[iJetColl].Jet_DeepFlavourCvsBDisc[JetInfo[iJetColl].nJet]  = DeepFlavourC/(DeepFlavourC + DeepFlavourB + DeepFlavourBB + DeepFlavourLepB);
+    //DeepFlavour discriminators
+    JetInfo[iJetColl].Jet_DeepFlavourBDisc[JetInfo[iJetColl].nJet]     = (DeepFlavourB != -1000 && DeepFlavourBB != -1000 && DeepFlavourLepB != -1000) ? DeepFlavourB + DeepFlavourBB + DeepFlavourLepB : -1000;
```

Comment:
@rauser A couple of comments:
   1) I do not see how we can get to -1000. If deepFlavour(Neg)JetTags_ is not set everything is defaulted to -10. I guess the case is if deepFlavour(Neg)JetTags_  are set, but not re-run. I would suggest a more uniform usage with DeepFlavourB < -5 picking every option. 
   2) If  DeepFlavourB is defaulted also all the others will be (unless we did something **really** stupid), to make the code more readable I would suggest using only the first to check for defaults.

Reply:
In the current implementation deepFlavour(Neg)JetTags_ are set, but not re-run. I see the advantage of the more uniform usage and implement it. PR should be updated by git magic :)